### DISCUSSION TITLE
Руководство ПЛИ, тело документа, 1.1. секции документа

### DISCUSSION THREAD

**Discussion by eugenekogan-spb** [fhir-ru/core#280] [Лабораторная РГ]
Информация в CDA документе "Протокол лабораторного исследования", в соответствии с данным руководством, должна быть разложена в ряд обязательных и опциональных разделов/секций. Для упрощения разбора документа и для лучшего представления секции должны быть расположены в указанном порядке.
<img width="620" alt="image" src="https://github.com/user-attachments/assets/5ecd475c-ee14-40e9-9ec4-0c55575815d4">
<img width="622" alt="image" src="https://github.com/user-attachments/assets/89c0e6b1-6186-4ea1-9879-7ebeea9b6a04">

**Comment by eugenekogan-spb** [comment]
1. Информация об исследованных материалах
У2-1:Элемент structuredBody ОБЯЗАН иметь один [1..1] элемент component, содержащий секцию "Информация об исследованных материалах", который:
1. ОБЯЗАН иметь один [1..1] элемент section, который:
1. ОБЯЗАН иметь один [1..1] элемент code (атрибут @code должен иметь значение "SPECIMENS", атрибут @codeSystem должен иметь значение "1.2.643.5.1.13.13.99.2.197", атрибут @codeSystemName должен иметь значение "Секции электронных медицинских документов", атрибут @displayName должен иметь значение "Информация об исследованных материалах", атрибут @codeSystemVersion должен быть представлен с не пустым значением), содержащий код секции;
2. ОБЯЗАН иметь один [1..1] элемент title, содержащий название секции, который должен иметь не пустое наполнение;
3. ОБЯЗАН иметь один [1..1] элемент text, содержащий человекочитаемое наполнение секции, который должен иметь не пустое наполнение.

**Reply by eugenekogan-spb** [reply]
```
* section
  * title = "Информация об исследованных материалах"
  * code
    * text = "SPECIMENS"
  * text
    * status =  #extensions 
    * div = "<div> Здесь человекочитаемое наполнение секции</div>"
```

**Reply by eugenekogan-spb** [reply]
Надо бы разобраться, как размещать в атрибуте div фактическое наполнение, приведенное в спецификации СЭМД. Там оно сделано в формате shtml.
@OlegPenzin

**Reply by OlegPenzin** [reply]
видимо, не хватило namespace для обозначения, что внутри div'a пойдет XHTML ( http://www.w3.org/1999/xhtml )
после этого можно вставлять все, что лежало внутри элемента section.text СЭМДа
```
* section
  * title = "Информация об оборудовании и расходных материалах"
  * code
    * text = "ANALYSERS"
  * text
    * status =  #extensions 
    * div = "<div xmlns=\"http://www.w3.org/1999/xhtml\"> <p>Здесь <b>человекочитаемое</b> <u>наполнение</u> секции</p></div>"
```

**Comment by eugenekogan-spb** [comment]
2. Информация оборудовании и расходных материалах

У2-2:Элемент structuredBody ОБЯЗАН иметь один [1..1] элемент component, содержащий секцию "Информация об использованном оборудовании и расходных материалах", который:
1. ОБЯЗАН иметь один [1..1] элемент section, который:
1. ОБЯЗАН иметь один [1..1] элемент code (атрибут @code должен иметь значение "ANALYSERS", атрибут @codeSystem должен иметь значение "1.2.643.5.1.13.13.99.2.197", атрибут @codeSystemName должен иметь значение "Секции электронных медицинских документов", атрибут @displayName должен иметь значение "Информация об использованном оборудовании и расходных материалах", атрибут @codeSystemVersion должен быть представлен с не пустым значением), содержащий код секции;
2. ОБЯЗАН иметь один [1..1] элемент title, содержащий название секции, который должен иметь не пустое наполнение;
3. ОБЯЗАН иметь один [1..1] элемент text, содержащий человекочитаемое наполнение секции, который должен иметь не пустое наполнение.

**Reply by eugenekogan-spb** [reply]
```
* section
  * title = "Информация об оборудовании и расходных материалах"
  * code
    * text = "ANALYSERS"
  * text
    * status =  #extensions 
    * div = "<div> Здесь человекочитаемое наполнение секции</div>"
```

**Comment by eugenekogan-spb** [comment]
3. Витальные параметры

У2-3:Элемент structuredBody МОЖЕТ иметь один [0..1] элемент component, содержащий секцию "Витальные параметры", который:
1. ОБЯЗАН иметь один [1..1] элемент section, который:
1. ОБЯЗАН иметь один [1..1] элемент code (атрибут @code должен иметь значение "VITALPARAM", атрибут @codeSystem должен иметь значение "1.2.643.5.1.13.13.99.2.197", атрибут @codeSystemName должен иметь значение "Секции электронных медицинских документов", атрибут @displayName должен иметь значение "Витальные параметры", атрибут @codeSystemVersion должен быть представлен с не пустым значением), содержащий код секции;
2. ОБЯЗАН иметь один [1..1] элемент title, содержащий название секции, который должен иметь не пустое наполнение;
3. ОБЯЗАН иметь один [1..1] элемент text, содержащий человекочитаемое наполнение секции, который должен иметь не пустое наполнение.

**Reply by eugenekogan-spb** [reply]
```
* section
  * title = "Витальные параметры"
  * code
    * text = "VITALPARAM"
  * text
    * status =  #extensions 
    * div = "<div> Здесь человекочитаемое наполнение секции</div>"
```

**Comment by eugenekogan-spb** [comment]
4. Результаты лабораторных исследований

У2-4:Элемент structuredBody ОБЯЗАН иметь один [1..1] элемент component, содержащий секцию "Результаты лабораторных исследований", который:
1. ОБЯЗАН иметь один [1..1] элемент section, который:
Руководство по реализации СЭМД "Протокол лабораторного исследования" в соответствии с третьим уровнем требований архитектуры клинических документов версии 2 (CDA 2.0 HL7 v.3)
      Страница 71
1. ОБЯЗАН иметь один [1..1] элемент code (атрибут @code должен иметь значение "RESLAB", атрибут @codeSystem должен иметь значение "1.2.643.5.1.13.13.99.2.197", атрибут @codeSystemName должен иметь значение "Секции электронных медицинских документов", атрибут @displayName должен иметь значение "Результаты лабораторных исследований", атрибут @codeSystemVersion должен быть представлен с не пустым значением), содержащий код секции;
2. ОБЯЗАН иметь один [1..1] элемент title, содержащий название секции, который должен иметь не пустое наполнение;
3. ОБЯЗАН иметь один [1..1] элемент text, содержащий человекочитаемое наполнение секции, который должен иметь не пустое наполнение.

**Reply by eugenekogan-spb** [reply]
```
* section
  * title = "Результаты лабораторных исследований"
  * code
    * text = "RESLAB"
  * text
    * status =  #extensions 
    * div = "<div> Здесь человекочитаемое наполнение секции</div>"
```

**Comment by eugenekogan-spb** [comment]
5. Оказанные услуги

У2-5:Элемент structuredBody МОЖЕТ иметь один [0..1] элемент component, содержащий секцию "Оказанные услуги", который:
1. ОБЯЗАН иметь один [1..1] элемент section, который:
1. ОБЯЗАН иметь один [1..1] элемент code (атрибут @code должен иметь значение "SERVICES", атрибут @codeSystem должен иметь значение "1.2.643.5.1.13.13.99.2.197", атрибут @codeSystemName должен иметь значение "Секции электронных медицинских документов", атрибут @displayName должен иметь значение "Оказанные услуги", атрибут @codeSystemVersion должен быть представлен с не пустым значением), содержащий код секции;
2. ОБЯЗАН иметь один [1..1] элемент title, содержащий название секции, который должен иметь не пустое наполнение;
3.МОЖЕТ иметь один [0..1] элемент text, содержащий человекочитаемое наполнение секции, который должен иметь не пустое наполнение.

**Reply by eugenekogan-spb** [reply]
```
* section
  * title = "Оказанные услуги"
  * code
    * text = "SERVICES"
  * text
    * status =  #extensions 
    * div = "<div> Здесь человекочитаемое наполнение секции</div>"
```

**Comment by eugenekogan-spb** [comment]
6. Связанные документы

У2-6:Элемент structuredBody МОЖЕТ иметь один [0..1] элемент component, содержащий секцию "Связанные документы", который:
1. ОБЯЗАН иметь один [1..1] элемент section, который:
1. ОБЯЗАН иметь один [1..1] элемент code (атрибут @code должен иметь значение "LINKDOCS", атрибут @codeSystem должен иметь значение "1.2.643.5.1.13.13.99.2.197", атрибут @codeSystemName должен иметь значение "Секции электронных медицинских документов", атрибут @displayName должен иметь значение "Связанные документы", атрибут @codeSystemVersion должен быть представлен с не пустым значением), содержащий код секции;
2. ОБЯЗАН иметь один [1..1] элемент title, содержащий название секции, который должен иметь не пустое наполнение;
3.МОЖЕТ иметь один [0..1] элемент text, содержащий человекочитаемое наполнение секции, который должен иметь не пустое наполнение.

**Reply by eugenekogan-spb** [reply]
```
* section
  * title = "Связанные документы"
  * code
    * text = "LINKDOCS"
  * text
    * status =  #extensions 
    * div = "<div> Здесь человекочитаемое наполнение секции</div>"
```
